js/index.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function() {
+
+	$('form').submit(addCity);
+
+	function addCity(event) {
+		event.preventDefault();
+		var city = $('#city-type').val();
+		console.log(city);
+
+		if (city === 'New York City' || city === 'NYC' || city === 'New York') {
+			$('body').addClass('nyc');
+			console.log('NYC Background')
+		}
+		if (city === 'LA' || city === 'LAX' || city === 'Los Angeles') {
+			$('body').addClass('la');
+			console.log('LA Background')
+		}
+		if (city === 'SF' || city === 'San Francisco' || city === 'Bay Area') {
+			$('body').addClass('sf');
+			console.log('SF Background')
+		}
+		if (city === 'Austin' || city === 'ATX') {
+			$('body').addClass('austin');
+			console.log('Austin Background')
+		}
+		if (city === 'Sydney' || city === 'SYD') {
+			$('body').addClass('sydney');
+			console.log('Sydney Background')
+		}
+	}
+	console.log('Changed background')
```

Comment:
Did you notice that this log statement ran right as you loaded the page and not after the background changed?

This is because everything that isn't _inside_ of the `addCity` function runs as soon as the `document` is `ready`. If you want something to run after the addCity function, it must be inside that function (and maybe at the end).